docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+
+= Configuring a run configuration
+
+You can configure a number of separate run configurations to run against each of your projects. Each configuration is saved automatically as you work.
+
+.Prerequisites
+
+* Install `mta-cli`.
+* Projects imported into {VscodePluginShortName}.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_MTA_extension_icon.png[MTA code extension]) on the Activity bar.
+. If you do not have the MTA CLI installed, click *Download* on the *Unable to find MTA CLI* window.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Optional: Enter a new *Name*.
+. At a minimum, complete the following configuration fields:
+
+* *Input*: Click *Add* and enter the input files or directories or click *Open File Explorer* and select the directory.
+* *Target*: Select one or more target migration paths.
+
+. Right-click the run configuration and select *Run*.
++
+When the analysis is complete, the Migration Toolkit for Applications extension displays a folder icon named *Report* and a directory that contains a per-file analysis of the project.
++
+. Click the *Report* icon.
++
+The MTA Report opens in your browser. From the report landing page you can navigate to detailed reports, such as Application Details, Issues, and Dependencies. The report is described in detail in link:{ProductDocUserGuideURL}/index#review_reports_cli-guide[{ProductShortName} Report] in the _{UserCLIBookName}_.
```

Comment:
"Application Details, Issues, and Dependencies" - no point in capitalizing in this context.